docs/source/user-guide/getting-started.rst

```diff
@@ -2,306 +2,141 @@
 
-This 20-minute guide to getting started with conda lets you try out
-the major features of conda. You should understand how conda works
-when you finish this guide.
+.. tip::
+
+   Anaconda Navigator is a graphical desktop application that enables you to use conda without having to run commands at the command line.
 
-SEE ALSO: `Getting started with Anaconda Navigator
-<https://docs.anaconda.com/anaconda/navigator/getting-started>`_, a
-graphical user interface that lets you use conda in a web-like interface
-without having to enter manual commands. Compare the Getting started
-guides for each to see which program you prefer.
+   See `Getting started with Anaconda Navigator <https://docs.anaconda.com/anaconda/navigator/getting-started>`_ to learn more.
 
 Before you start
 ================
 
-You should have already `installed
-Anaconda <https://docs.anaconda.com/anaconda/install/>`_.
-
-Contents
-========
-
- - :ref:`Starting conda <starting-conda>` on Windows, macOS, or Linux. 2 MINUTES
-
- - :ref:`Managing conda <managing-conda>`. Verify that Anaconda is installed and check that conda is updated to the current version. 3 MINUTES
-
- - :ref:`Managing environments <managing-envs>`. Create :doc:`environments <../user-guide/concepts/environments>` and move easily between them.  5 MINUTES
-
- - :ref:`Managing Python <managing-python>`. Create an environment that has a different version of Python. 5 MINUTES
-
- - :ref:`Managing packages <managing-pkgs>`. Find packages available for you to install. Install packages. 5 MINUTES
-
-TOTAL TIME: 20 MINUTES
-
-
-.. _starting-conda:
+You should have already installed conda before beginning this getting started guide. Conda can be found in many distributions, like `Anaconda Distribution <https://docs.anaconda.com/free/anaconda/install/>`_, `Miniconda <https://docs.conda.io/projects/miniconda/en/latest/>`_ or `Miniforge <https://github.com/conda-forge/miniforge>`_.
 
 Starting conda
 ==============
 
-**Windows**
-
-* From the Start menu, search for and open "Anaconda Prompt."
-
-.. figure:: /img/anaconda-prompt.png
-   :width: 50%
-
-   ..
-
-|
-
-On Windows, all commands below are typed into the Anaconda Prompt window.
-
-**MacOS**
-
-* Open Launchpad, then click the terminal icon.
-
-On macOS, all commands below are typed into the terminal window.
+Conda is available on Windows, macOS, or Linux and can be used with any terminal application (or shell) in macOS/Linux, while in Windows, we recommend using the Anaconda Prompt application provided by the Anaconda Distribution or Miniconda installations.
 
-**Linux**
+.. tab-set::
 
-* Open a terminal window.
+   .. tab-item:: Windows
 
-On Linux, all commands below are typed into the terminal window.
+      #. From the Start menu, search for "Anaconda Prompt".
+      #. Open the Anaconda Prompt desktop app that appears in the search sidebar.
```

Comment:
```suggestion
      #. Open either cmd.exe or Powershell to open a terminal to use conda
```

I would like to not have Anaconda specific instructions here. This should be a valid way to open a terminal window to use conda with. I tested this out on a VM.